content/_data/changelogs/lts.yml

```diff
@@ -2837,6 +2837,143 @@
         These were made to anticipate a bugfix in Firefox which has been backed out since.
         (regression in 2.164.3)
 
+- version: 2.204.1
+  date: 2019-12-18
+  lts_predecessor: 2.190.3
+  lts_baseline: '2.204'
+  changes: # compared to lts_baseline
```

Comment:
https://github.com/jenkinsci/jenkins/pull/4208 from from 2.198 is missing. I would recommend to add it and to document in upgrade guidelines (there are agent upgrade guidelines somewhere)

Reply:
Still need to find and add agent upgrade guidelines or link to them.

Referred to #4208 in https://github.com/jenkins-infra/jenkins.io/pull/2720/commits/5c846293873eaa876ca7311459ffd050f76bd728